sroka/api/s3_connection/README.md

```diff
@@ -33,4 +33,31 @@ s3_download_data('s3://bucket/folder/2019_01_01/', output_file='./clicked.csv',
 
 s3_download_data('s3://bucket/folder/2019_01_01/part-111-111-111-111-111-111.csv', 
    output_file='./clicked_1.csv')
-```
\ No newline at end of file
+```
+
+
+### `s3_upload_data(data, bucket, path, sep=',')`
```

Comment:
This file could be auto-generated based on docstrings using [pydoc-markdown](https://pypi.org/project/pydoc-markdown/).